source/includes/_contact_interactions.md

```diff
@@ -0,0 +1,297 @@
+## Get Contact Interactions for an Advertiser
+
```

Comment:
This endpoint supports integration with CRM products. It returns all interaction by a contacts (aka leads)for an advertiser in a given time period. It requires the the following parameters for a successful response Gmaid (advertise ID), created_before date and created_after_date.
The response is organized by campaigns with the interactions/leads nested within. Please note that the response includes the associated contact information and its associated meta data.
The recommended usage is to call the Contacts endpoint to seed your CRM with contacts followed by a call to the contacts interactions endpoint with the exact time period to receive the interactions/leads

Reply:
Please note that the contact information in this endpoint is for linking the interactions/leads with the contact. Importing them into your CRM from this endpoint will cause duplicates.